### PR TITLE
WIP: feat: Contribution guide for open-source projects

### PR DESCRIPTION
**Pull request by fluorumlabs**

### CODE REVIEW

File: articles/guide/contributing/index.asciidoc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+title: Contributing to an Open-Source Projects
```
Comment (jouni) [inline articles/guide/contributing/index.asciidoc:2]:
```suggestion
title: Contributing to Open-Source Projects
```

File: articles/guide/contributing/overview.asciidoc [hunk 2]
```diff
@@ -0,0 +1,224 @@
+---
+title: Contributing to an Open-Source Projects
```
Comment (jouni) [inline articles/guide/contributing/overview.asciidoc:2]:
```suggestion
title: Overview
```

File: articles/guide/contributing/overview.asciidoc [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+order: 1
+layout: page
+---
+
+:experimental:
+:commandkey: &#8984;
+
+= Contributing to an Open-Source Projects
```
Comment (jouni) [inline articles/guide/contributing/overview.asciidoc:10]:
```suggestion
= Contributing to Open-Source Projects
```

File: articles/guide/contributing/overview.asciidoc [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+Straight up crashes and lockups are pretty convincing, but not all bugs are that blatant. 
+Even if the problem was spotted during code review, describe the impact you think it can have on users. 
+
+Describe the solution::
+
+Once the problem is established, describe what you are actually doing about it in technical detail.  It's important to describe the change in plain English for the reviewer to verify that the code is behaving as you intend it to.
+
+Solve only one problem per patch::
+
+If your description starts to get long, that's a sign that you probably need to split up your patch. See “Only one logical change per patch”.
```
Comment (martaolszewskatusiek) [inline articles/guide/contributing/overview.asciidoc:63]: There should probably be: [See "Separate Your Changes"] instead of [See “Only one logical change per patch”]

File: articles/guide/contributing/overview.asciidoc [hunk 5]
```diff
@@ -0,0 +1,227 @@
...
++
+--
+```
+feat: Create a Valo icon font for icons in Valo
+```
+
+Start with a good subject message in imperative form with 50 chars or less. 
+A properly formed Git commit subject line should always be able to complete the following sentence:
+
+If applied, this commit will <__your subject line here__>
```
Comment (martaolszewskatusiek) [inline articles/guide/contributing/overview.asciidoc:142]: This is also mentioned in chunk "Describe your changes". Maybe can be dropped in the previous section? Or is it better t repeat, if important?

**Review by jouni — APPROVED** [review]
